odoo/addons/base/tests/test_orm.py

```diff
@@ -35,10 +35,6 @@ def test_access_deleted_records(self):
         # Deleting an already deleted record should be simply ignored
         self.assertTrue(p1.unlink(), "Re-deleting should be a no-op")
 
-        # Updating an already deleted record should raise, even as admin
-        with self.assertRaises(MissingError):
-            p1.write({'name': 'foo'})
-
```

Comment:
Wait a minute, this test is meaningful.
I suspect that a `flush()` should be added to make it 100% correct, though.

Reply:
Since the test is actually testing  `read` implicitly instead of `write`, it is not obvious if `flush` is missing, and adding it without any other change would have no effect since it is already raising without it.